core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryEdit2Presenter.kt

```diff
@@ -8,6 +8,7 @@ import com.ustadmobile.core.networkmanager.defaultHttpClient
 import com.ustadmobile.core.util.MessageIdOption
 import com.ustadmobile.core.util.UMFileUtil
 import com.ustadmobile.core.util.UMUUID
+import com.ustadmobile.core.util.ext.convertToJsonObject
```

Comment:
Issue spotted on this presenter that is not directly video related, but related to importing from a url:

handleClickSave's post call to the server needs to be put in a try/catch clause. It is running on the main dispatcher. Any exception will crash the whole app. If there was a connection error or such, the entire app would crash here.

If we get an exception, or a non-200 response status, we should use the snack bar to tell the user that an error happened. In this event, we shouldn't call finish.